base/src/com/google/idea/blaze/base/qsync/BazelDependencyBuilder.java

```diff
@@ -207,14 +213,14 @@ public OutputInfo build(
         ThrowOption.ALLOW_PARTIAL_SUCCESS,
         ThrowOption.ALLOW_BUILD_FAILURE);
 
-      return createOutputInfo(outputs, buildDependenciesFlags.requestedOutputGroups(), buildTime, context);
+      return createOutputInfo(outputs, buildDependenciesBazelInvocationInfo.requestedOutputGroups(), buildTime, context);
     }
   }
 
   @VisibleForTesting
-  public BuildDependenciesBazelInvocationInfo prepareAspectAndGetInvocationFlags(BlazeContext context,
-                                                                                 Set<Label> buildTargets,
-                                                                                 Set<QuerySyncLanguage> languages)
+  public BuildDependenciesBazelInvocationInfo getInvocationInfo(BlazeContext context,
+                                                                Set<Label> buildTargets,
+                                                                Set<QuerySyncLanguage> languages) {
     throws IOException, BuildException {
```

Comment:
Can we drop the `throws` here too?
<img width="1444" alt="Screenshot 2025-03-07 at 12 24 59 PM" src="https://github.com/user-attachments/assets/21d0adce-add1-485e-9357-fd51cfadead8" />